metadata-ingestion-modules/airflow-plugin/tests/integration/dags/dag_to_filter_from_ingestion.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+
+from datahub_airflow_plugin.entities import Dataset, Urn
+
+with DAG(
+    "dag_to_filter_from_ingestion",
```

Comment:
the previous name was too verbose
```suggestion
    "dag_to_skip",
```